### ISSUE TITLE
problem of fine-tuning's learning rate and batch-size

### ISSUE THREAD

**Issue by caopulan** [pengzhiliang/MAE-pytorch#10]
https://github.com/pengzhiliang/MAE-pytorch/blob/main/files/pretrain_base_0.75_400e_finetune_100e.txt
It can be seen that learning rate reaches 4e-3 after warm-up.  Did you set 4096 batch-size during fine-tuning?

**Comment by pengzhiliang** [comment]
According to Appendix.1, base lr×batchsize / 256.
In end-to-end finetuning, and table 8 shows base lr = 1e-3.
